sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md

```diff
@@ -6,19 +6,25 @@ We use [NUnit 3][nunit] as our testing framework.
 
 [Azure.Core.TestFramework's testing framework][core_tests] provides a set of reusable primitives that simplify writing tests for new Azure SDK libraries.
 
+The installation of `Test proxy` is a prerequisite for running the tests. Please run the `test-proxy --version` to ensure that test proxy is installed on your computer. If it is not installed, Please refer to [this document](https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md#installation) for the installation of the test proxy.
+
 ## Testing project structure
 
 With the help of [Azure.ResourceManager.Template][mgmt_template], the basic testing project and file structure(see following for details) will be generated under `sdk\<service name>\Azure.ResourceManager.<service>\tests` directory.
 
 ```text
+sdk\<service name>\Azure.ResourceManager.<service>\assets.json
```

Comment:
The folder structure has been updated in the template already?

Reply:
yes, on this pr i updated the template